test/integration/create_users_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class CreateUsersTest < ActionDispatch::IntegrationTest
+  def setup
+    @user = users(:valid)
+  end
+
+  test "can create a user" do
```

Comment:
Could you use `it` blocks instead of `test`. Might mean also switching `setup` to `before` blocks. It's a pattern we're migrating towards at Omise